### DISCUSSION TITLE
Clients throught WIREGUARD

### DISCUSSION THREAD

**Discussion by Atorcha** [AdguardTeam/AdGuardHome#7192] [Q&A]
![Captura de pantalla 2024-08-12 103256](https://github.com/user-attachments/assets/a18b8b4e-3dd9-46f6-98da-cb898dbb7679)
![IMG_0039](https://github.com/user-attachments/assets/bdc7e75b-3d42-496d-be73-939283bffb6d)

Hello. How can I see the clients connected trought wireguard (192.168.10.101 for example) and not IP of WG (10.6.0.10)

I need to aply filters and rules for that devices!

Thanks

**Comment by ppfeufer** [comment]
That's a VPN feature called DNS Masquerading. It's essentially what a VPN does, hide the actual client.

**Reply by Atorcha** [reply]
![IMG_0042(1)](https://github.com/user-attachments/assets/dbf12db7-7d68-487c-b58d-c2a591baafb7)
![IMG_1922](https://github.com/user-attachments/assets/c18009e9-da52-4a10-b82f-49cd3b244a2b)
 But I have disable this option
